htlcswitch/link_test.go

```diff
@@ -1686,6 +1686,9 @@ func updateState(batchTick chan time.Time, link *channelLink,
 // sleep in this test and the one below
 func TestChannelLinkBandwidthConsistency(t *testing.T) {
 	t.Parallel()
+	if !hodl.DebugBuild{
+		t.Skipf("test must be run with '-tags debug")
```

Comment:
I would rather make all these `t.Fatalf`, as now a test regression could go unnoticed.

Reply:
Why would you like this to be t.Fatalf if you know that it will fail for sure?

There is no question about it. When running COVERS , tags=debug is not provided to "go run" and these 3 tests fail. (see PR description).   And they fail without anyone noticing so making it Fatalf will not make any change.

The reason we don't notice has to do with #1593 . Once this is accepted the Travis check will be red we will need to handle it. Since there is no error here, handling would be to use skip.

Reply:
Ok, that wasn't made clear from the PR description. 

Is it possible to instead run Cover with `tags=debug`?

Reply:
@halseth I guess this is possible and it will solve these 3 failures.  In that case, I can change the t. Skipf to t.Fatalf in all places.  

But, is this something you want?  Do you really want to run Cover with debug build???

Reply:
Dunno, would that cause issues?

cc @cfromknecht

Reply:
@halseth It should not cause issues. Still, if the intention is that production LND will not be compiled with tags=debug, coverage should also be checked without it, IMHO. (I would also consider if integration tests should use debug build).

Let me know what you want and I will change it accordingly.

Reply:
@halseth - any decision?

Reply:
For now I think using a debug build is the right direction.

Reply:
Sure, as far as I'm concerned, you can close this one and open a new task for modifying the Makefile to use debug build for Coverage. 

Thanks